packages/SystemsThinkingWorkbench-Core.package/STWConnection.class/class/vertices.color.borderWidth.borderColor..st

```diff
@@ -0,0 +1,3 @@
+instance creation
+vertices: verts color: c borderWidth: bw borderColor: bc
```

Comment:
But could you not call initialize form the STWConnections own `from:to:color:width:` method, as in
```
^ (self
 		vertices: {startPoint. endPoint}
 		color: Color black
 		borderWidth: lineWidth
 		borderColor: lineColor) beStraightSegments; yourself
```